docker-compose.yaml

```
@@ -35,18 +33,24 @@ services:
       - ./frontend:/app
     environment:
       - API_BASE_URL=https://api.zeno-staging.ds.io/
-      - LOCAL_API_BASE_URL=http://api:8000/
+      - LOCAL_API_BASE_URL=http://api:8000
       - STREAMLIT_URL=${STREAMLIT_URL:-http://localhost:8501}
 
   langfuse-server:
     image: langfuse/langfuse:2
     depends_on:
-      db:
-        condition: service_healthy
+      migrate:
+        condition: service_completed_successfully
     ports:
       - "3000:3000"
-    #env_file:
-    #  - .env
+    healthcheck:
+      # Uses wget because curl is not available in the langfuse image
+      test: ["CMD", "wget", "-qO-", "http://172.20.0.3:3000/api/public/ready"]
```

this internal ip might not be stable. Should we use localhost or the service name instead?
```suggestion
      test: ["CMD", "wget", "-qO-", "http://localhost:3000/api/public/ready"]
```